### PR TITLE
MULTIARCH-4704: Upadate tekton branch name

### PR DESCRIPTION
**Pull request by AnnaZivkovic**
Adjust the .tekton files  in order to make the pipelines run and target the right components

### CODE REVIEW

**Comment by AnnaZivkovic** [comment]
We want to keep the application name as multiarch-tuning-operator as that is where components for the bundle and operator are located

**Comment by aleskandro** [comment]
> We want to keep the application name as multiarch-tuning-operator as that is where components for the bundle and operator are located

We can keep the same components name, but the application will have a different name.

The current one, `multiarch-tuning-operator` is bound to the `main` branch. 
I expect the ProjectDevelopmentStreamTemplate can name a version-based one like `multiarch-tuning-operator-v0-9`, `multiarch-tuning-operator-v1-0`, ...

See how I did a similar work for the FBCs (following the OCP versioning)

File: .tekton/multiarch-tuning-operator-push.yaml [hunk 1]
```diff
@@ -7,11 +7,11 @@ metadata:
...
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression:
-      event == "push" && target_branch == "main" && (".tekton/multiarch-tuning-operator-pull-request.yaml".pathChanged() || "apis/***".pathChanged() || "controllers/***".pathChanged() || "pkg/***".pathChanged() || "test/***".pathChanged() || "konflux.Dockerfile".pathChanged() || "go.mod".pathChanged() || "main.go".pathChanged() || "go.sum".pathChanged() || "trigger-konflux-builds.txt".pathChanged() )
+      event == "push" && target_branch == "v0.9" && (".tekton/multiarch-tuning-operator-pull-request.yaml".pathChanged() || "apis/***".pathChanged() || "controllers/***".pathChanged() || "pkg/***".pathChanged() || "test/***".pathChanged() || "konflux.Dockerfile".pathChanged() || "go.mod".pathChanged() || "main.go".pathChanged() || "go.sum".pathChanged() || "trigger-konflux-builds.txt".pathChanged() )
   creationTimestamp: null
   labels:
-    appstudio.openshift.io/application: multiarch-tuning-operator
-    appstudio.openshift.io/component: multiarch-tuning-operator
+    appstudio.openshift.io/application: multiarch-tuning-operator-v0.9
```
Comment (aleskandro) [inline .tekton/multiarch-tuning-operator-push.yaml]:
```suggestion
    appstudio.openshift.io/application: multiarch-tuning-operator-v0-9
```

Shouldn't this be v0-9? I believe Tekton will complain about dots in the names.
Similar to https://github.com/openshift/multiarch-tuning-operator/blob/fbc-4.16/.tekton/fbc-v4-16-pull-request.yaml

Comment (AnnaZivkovic) [reply .tekton/multiarch-tuning-operator-push.yaml]: I think you are right according to the description in the ProjectDevelopmentStreamTemplate

File: .tekton/multiarch-tuning-operator-bundle-pull-request.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ spec:
     - name: image-expires-after
       value: 5d
     - name: output-image
-      value: quay.io/redhat-user-workloads/multiarch-tuning-ope-tenant/multiarch-tuning-operator-v0-9/multiarch-tuning-operator-bundle-v0-9:on-pr-{{revision}}
+      value: quay.io/redhat-user-workloads/multiarch-tuning-ope-tenant/multiarch-tuning-operator/multiarch-tuning-operator-bundle:on-pr-{{revision}}
```
Comment (aleskandro) [inline .tekton/multiarch-tuning-operator-bundle-pull-request.yaml]: Let's confirm this image name once the internal PR lands.

**Comment by aleskandro** [comment]
/lgtm
/approved
/hold

(feel free to unhold if the jobs pass)

**Comment by aleskandro** [comment]
/override "Red Hat Konflux / multiarch-tuning-operator-0-9-enterprise-contract / multiarch-tuning-operator-0-9" "Red Hat Konflux / multiarch-tuning-operator-0-9-enterprise-contract / multiarch-tuning-operator-bundle-0-9"

unrelated ot this pr

**Pull request merged**